main.go

```diff
@@ -2,16 +2,26 @@ package main
 
 import (
 	"flag"
+	"fmt"
+	"os"
+
 	log "github.com/Sirupsen/logrus"
 	"github.com/rancherio/go-machine-service/events"
 	"github.com/rancherio/go-machine-service/handlers"
-	"os"
+)
+
+var (
+	VERSION   = "v0.11.0"
```

Comment:
VERSION should be there, for a few reasons. One, its easier to track down than gitcommits. Two, when we build from non-version controlled sources (for example by downloading the source from the releases page), we won't be able to see the gitcommits. 

As long as we create a PR before every release (which is already a manual step),  we should be fine.